eg/gps-hardware-serial.js

```diff
@@ -0,0 +1,18 @@
+var five = require("../lib/johnny-five.js");
+var board = new five.Board();
+
+board.on("ready", function() {
+
+  /*
+   * Explicitly setting HW_SERIAL1 for the port
+   */
+  var gps = new five.GPS({
+    port: this.io.SERIAL_PORT_IDs.HW_SERIAL1
```

Comment:
I'm not going to block on this for now, but I'm going to think about how we can make this less icky